pkg/util/util.go

```diff
@@ -55,3 +56,9 @@ func (dt *ThroughputCalc) Start() {
 		}()
 	}
 }
+
+// MaskPassword is used to mask sensitive password data before outputing to persistent stream like logs.
+func MaskPassword(s string) string {
```

Comment:
Should this be called in our Logger on every log or is that too expensive? Are we sure we caught all places that passwords can leak?

Reply:
I went through the code and added masking to all places that could leak. We could add this to all logs, not sure about the perf penalty. I could do some benchmarks to confirm. Might not be worth the trouble.

What do you think? @cevian

Reply:
Ok, yeah lets leave as-is then.